prestodb/src/test/java/io/pravega/connectors/presto/util/MockSchemaRegistryClient.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.connectors.presto.util;
+
+import io.pravega.schemaregistry.client.SchemaRegistryClient;
+import io.pravega.schemaregistry.client.exceptions.RegistryExceptions;
+import io.pravega.schemaregistry.contract.data.CodecType;
+import io.pravega.schemaregistry.contract.data.Compatibility;
+import io.pravega.schemaregistry.contract.data.EncodingId;
+import io.pravega.schemaregistry.contract.data.EncodingInfo;
+import io.pravega.schemaregistry.contract.data.GroupHistoryRecord;
+import io.pravega.schemaregistry.contract.data.GroupProperties;
+import io.pravega.schemaregistry.contract.data.SchemaInfo;
+import io.pravega.schemaregistry.contract.data.SchemaWithVersion;
+import io.pravega.schemaregistry.contract.data.VersionInfo;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * this class has a very limited use in unit testing, so many methods not implemented
```

Comment:
It is not clear to me if there is some follow-up work that needs to be done here. Should we have an issue to implement what is missing?

Reply:
this is just class in Test build.  imho nothing to do.  it implements what we need for the test.  if anyone adds another test that happens to need another method that is unimplemented they could add it.

another backlogged PR adds and uses EmbeddedSchemaRegistry

https://github.com/pravega/presto-connector/pull/42/files#diff-8a6311f05fe2460ebdd74536fabc4437f9bb20531b4d04b2649dc2b0ae857aa6

EmbeddedSchemaRegistry would obsolete this class here.